src/test/java/ca/corefacility/bioinformatics/irida/service/export/NcbiExportSubmissionServceTest.java

```diff
@@ -7,6 +7,7 @@
 
 import javax.validation.Validator;
```

Comment:
Can you please fix the name of this class

Reply:
Renamed in 9d2fd2f7a9

Reply:
Thanks